### ISSUE TITLE
[BUG]: Android: E/Unity: NullReferenceException: Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by ammyt** [Unity-Technologies/UnityRenderStreaming#703]
### Package version

3.1.0-exp.3

### Environment

```markdown
* OS: Android 11
* Unity version: 2021.3
* Graphics API:
* Browser:
```

### Steps To Reproduce

Build for Android, install app then add 192.168.0.42:80 as Address then click on "Bidirectional".
Clicking on Setup produces this error, and on Hangup as well.

Cannot connect to a another Unity on PC, I suspect it relates to this error.

![image](https://user-images.githubusercontent.com/24976090/174867024-e4b6f690-b1e5-4e84-a477-8c5dcea53409.png)

**Comment by kannan-xiao4** [comment]
@ammyt 
Thank you for your reporting. We will check this issue.
memo: URS-463

**Comment by karasusan** [comment]
@ammyt
We released the latest version 3.1.0-exp.5 this month, could you try that?
